shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/resources/KNativeBrokerConfigurationSpec.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.service.smartevents.shard.operator.v2.resources;
+
+public class KNativeBrokerConfigurationSpec {
+
+    String topicName;
```

Comment:
Personally I'd move topicName, numberOfPartitions,numberOfReplicas into the KafkaConfiguration.